ImageFeed/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+
+class ProfileViewController: UIViewController {
+    private var profileImageView: UIImageView = {
+        let image = UIImage(systemName: "person.crop.circle.fill")
+        let imageView = UIImageView(image: image)
+        imageView.tintColor = .gray
+        return imageView
+    }()
+    
+    private var profileNameLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Екатерина Новикова"
+        label.font = UIFont.systemFont(ofSize: 23, weight: .bold)
+        label.textColor = UIColor(named: "YP White")
+        return label
+    }()
+    
+    private var profileLinkLabel: UILabel = {
+        let label = UILabel()
+        label.text = "@ekaterina_nov"
+        label.font = UIFont.systemFont(ofSize: 13)
+        label.textColor = UIColor(named: "YP Grey")
+        return label
+    }()
+    
+    private var profileDescriptionLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Hello, world!"
+        label.font = UIFont.systemFont(ofSize: 13)
+        label.textColor = UIColor(named: "YP White")
+        return label
+    }()
+    
+    private var logoutButton: UIButton = {
+        let image = UIImage(systemName: "ipad.and.arrow.forward")!
+        let button = UIButton.systemButton(
+            with: image,
+            target: self,
+            action: #selector(logOutButtonTapped)
+        )
+        button.tintColor = UIColor(named: "YP Red")
+        return button
+    }()
```

Comment:
передавать ссылку на себя в кнопку – не очень классно, Xcode даже ругается. В таких случаях лучше сделать lazy свойсвто
```suggestion
    private lazy var logoutButton: UIButton = {
        let image = UIImage(systemName: "ipad.and.arrow.forward")!
        let button = UIButton.systemButton(
            with: image,
            target: self,
            action: #selector(logOutButtonTapped)
        )
        button.tintColor = UIColor(named: "YP Red")
        return button
    }()
```